src/Claims/Claim.php

```diff
@@ -98,15 +93,37 @@ public function getName()
     }
 
     /**
-     * Validate the Claim value.
+     * Validate the claim in a standalone Claim context.
      *
      * @param  mixed  $value
      *
      * @return bool
      */
-    public function validate($value)
+    public function validateCreate($value)
+    {
+        return $value;
+    }
+
+    /**
+     * Validate the Claim within a Payload context.
+     *
+     * @return bool
+     */
+    public function validatePayload()
+    {
+        return $this->getValue();
+    }
+
+    /**
+     * Validate the Claim within a refresh context.
+     *
+     * @param  int  $refreshTTL
+     *
+     * @return bool
+     */
+    public function validateRefresh($refreshTTL)
```

Comment:
Is every possible refresh flow going to use `$refreshTTL` and not need something else? If there are other possibilities, I'd rather leave out the argument and do something more generic to get those kinds of values. Maybe inject the `Factory` instance in the `Claim` constructor, and have a general `Factory::getSetting` method providing access to configured values like `ttl` & `refreshTTL`.

Of course, I'm not sure if there _are_ legitimate cases for needing other parameters in the refresh flow. Even if you have the chaining function differently, it would still be based on a time delta. In #735 I mentioned a possible `RefreshCount` claim that would need a `refresh_limit` value, but that's a fringe idea and I'm not actually certain why anyone would want that limitation over a timed one. (Though I believe it was requested in the issues here once before)